annotation/data/generations-gpt2-medium-nyt-dev.json

```diff
@@ -0,0 +1,3804 @@
+{
```

Comment:
I think we want to avoid putting these large generation files in the git repository, especially since they are already on the GCP bucket. Could you maybe add a .gitignore for your data directory?

You can mention in the README that people can wget these files from gs://... if you'd like.

Reply:
I actually added that to readme in branch #113 and yeah agreed that all the raw generation files shouldn't be in here

Reply:
Agreed. Will add a `/raw_data` folder to the gitignore.